pages/docs/platform/webhooks.mdx

```diff
@@ -85,7 +85,7 @@ Transforms are defined as simple JavaScript functions that accept three argument
     The raw JSON payload from the POST request body
   </Property>
   <Property name="headers" type="object" attribute>
-    A map of HTTP headers sent along with the request as key-value pairs.
+    A map of HTTP headers sent along with the request as key-value pairs. Header names are standardized by making the first character and any characters following a hyphen uppercase and the rest lowercase. For more details, [check out](https://pkg.go.dev/net/http#CanonicalHeaderKey) the underlying implementation reference.
```

Comment:
@BrunoScheufler: let's call out explicitly that they are case sensitive?

Reply:
good point